File: python/image/show.py
import cv2, sys

img = cv2.imread(sys.argv[1], cv2.IMREAD_COLOR)
cv2.namedWindow("test", cv2.WINDOW_AUTOSIZE)
cv2.imshow("test", img)
cv2.waitKey(5000)
#cv2.destroyWindow("test")
cv2.destroyAllWindows()

File: cpp/image/show.cpp
#include <cv.h>
#include <highgui.h>

using namespace cv; 

int main(int argc, char *argv[])
{
    Mat image; 
    image=imread(argv[1]); 
    namedWindow("test"); 
    imshow("test", image); 
    waitKey(5000); 
    return 0; 
}
